app/controllers/api_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ApiController < UnauthenticatedApiController
-    @current_user = nil
+    before_action :authenticate_v1_user!
+
+    protected
+  
+    def current_user
+      @current_user = current_v1_user
+    end
```

Comment:
Files should have an empty line at the end of them

Reply:
biniuca